.dockerignore

```diff
@@ -1,3 +1,4 @@
 support
 _build
 _sanitize
+build*/
```

Comment:
nit: I think you can just have `build` and not the `*/`?

Reply:
I sometimes have directories like build-aarch64 and build-mingw